server/src/messages/messages.service.ts

```diff
@@ -6,12 +6,13 @@ import { User } from '../acccount-manager/entities/user.entity';
 import { Message } from './entities/message.entity';
 import { CreateMessageDto } from './dto/create-message.dto';
 import { UpdateMessageDto } from './dto/update-message.dto';
+import { ReturnUserDto } from '../acccount-manager/dto/auth.dto';
 
 @Injectable()
 export class MessagesService {
   constructor(@InjectRepository(Message) private messagesRepository: Repository<Message>) {}
 
-  async create(createMessageDto: CreateMessageDto, user: User): Promise<Message> {
+  async create(createMessageDto: CreateMessageDto, user: ReturnUserDto): Promise<Message> {
```

Comment:
I think we should be able to keep the User type here, since it's a service.

Reply:
@esteban-gs, I've reverted my change so it uses `User` type, but I am curious about the mismatch in the types  of params we're using for this service, `CreateMessageDto` and `User`.  My initial thought was that using `dtos` would be more inline with how data is moved around the application, but I'm new to Nest.js and so might not be understanding this correctly.